cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/ConfigurationTable.java

```diff
@@ -110,7 +109,7 @@ public void write(Type type, CConfiguration conf) throws IOException {
         try {
           table.close();
         } catch (IOException ioe) {
-          LOG.error("Error closing HTable for " + tableName, ioe);
+          LOG.error(String.format("Error closing HTable for %s:%s", Constants.SYSTEM_NAMESPACE, TABLE_NAME), ioe);
```

Comment:
This class was using Apache Commons Logging, so I think this was needed. Switching it to use slf4j.